components/browser/menu/src/main/res/values/style.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <style name="mozacBrowserMenuItemDividerHorizontal">
```

Comment:
My gut feeling is that this should start with an uppercase character (`Mozac`). Is there a style guide for styles?

Reply:
You're right, I'll fix it.

> Is there a style guide for styles?
No that I'm aware of, but I was checking the Android ones and they follow the upper case pattern.

Reply:
Looking at this it seems like we want `Mozac.Browser.Menu.Item.Divider.Horizontal`? :)

Reply:
Thanks for keep asking. I learned something new :)

I took for granted that it wasn't possible to do that. Without having something like this:
![image](https://user-images.githubusercontent.com/773158/51207587-3e145300-18d9-11e9-80cd-4dede2379d96.png)

Because the way that styles work, it's that every time that you have a dot in a new style name you are inheriting from it. (For this reason, every style that you are referring with a dot should it should exist before using it)

![image](https://user-images.githubusercontent.com/773158/51207853-0063fa00-18da-11e9-885f-8cb4f93c0316.png)

To have a good answer for you, I kept searching and I found [themes_base.xml,](https://github.com/aosp-mirror/platform_frameworks_support/blob/master/v7/appcompat/res/values/styles_base.xml) the namespace for *Base.AppCompat.**.

When they want to define a new style without having a style for every dot they use `parent=""`
![image](https://user-images.githubusercontent.com/773158/51208321-4a011480-18db-11e9-9011-9f0be8cd96fa.png)

Now we can have :) 

![image](https://user-images.githubusercontent.com/773158/51208353-7157e180-18db-11e9-9801-7940fc135b5f.png)

A good resource about [Style and themes.](https://guides.codepath.com/android/Styles-and-Themes)